settings.php

```diff
@@ -247,10 +250,4 @@
             }
         }
     }
-
-    $settings->add(new admin_setting_heading('tool_objectfs/testsettings',
```

Comment:
is this lang string no longer used now?

Reply:
Ah yes it does, updated to remove the lang string